Mathlib/Algebra/Category/ModuleCat/Monoidal/Closed.lean

```diff
@@ -57,7 +57,6 @@ theorem ihom_map_apply {M N P : ModuleCat.{u} R} (f : N ⟶ P) (g : ModuleCat.of
 
 open MonoidalCategory
 
--- Porting note: `CoeFun` was replaced by `DFunLike`
 -- I can't seem to express the function coercion here without writing `@DFunLike.coe`.
```

Comment:
Is this not part of the porting note?

Reply:
Note that this `coe` can go away after #19511.

Reply:
oh, I didn't think of those two comments as being connected, but I guess maybe they were. In any case, it can go away soon.